framework/src/eventloop/eventloop_async.c

```diff
@@ -183,6 +181,24 @@ static void eventloop_thread_safe_cb_destroy(thread_safe_cb_t *thread_safe_cb)
 	EL_FREE(thread_safe_cb);
 }
 
+void eventloop_unregister_thread_safe_cb(el_async_t *handle)
+{
+
+	int thread_safe_cb_list_idx;
+	thread_safe_cb_t *curr;
+
+	thread_safe_cb_list_idx = THREAD_SAFE_CB_LIST_IDX_HASH(getpid());
+	while ((curr = (thread_safe_cb_t *)sq_remfirst(&CB_LIST(thread_safe_cb_list_idx)))) {
+		curr->func_handle->refs--;
+		if (curr->func_handle->refs == 0) {
+			eventloop_func_handle_remove(curr->func_handle);
+		}
+		EL_FREE(curr);
+	}
+
```

Comment:
May I ask one thing that is uv_async_deinit(loop, ASYNC_HANDLE(thread_safe_cb_list_idx)) required here for safety?